### PR TITLE
default order of top sales, then a filter endpoint

### PR DESCRIPTION
**Pull request by Dev-wonderful**
## Default sort order should be by top sales
​
## Description

* I created a function to query and arrange the shops in the order of highest sales
* It was fix for the initial setup which was to arrange by newest as the default
* I also created a route for filtering based on 3 things, newest, oldest and active status
* A different endpoint was used so as not to overload the `shop/all` endpoint with a lot of functions
​
## Related Issue (Link to linear ticket)

​
## Motivation and Context

Our PM said highest sales should be the default order
​
## How Has This Been Tested?

Thunderclient request
​
## Screenshots (if appropriate - Postman, etc):
​
Screenshot (236)
Screenshot (238)
Screenshot (239)
Screenshot (240)
Screenshot (241)
Screenshot (237)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Dev-wonderful** [comment]
@zxenonx

**Review by Freeman-kuch — APPROVED** [review]

File: utils.py [hunk 1]
```diff
@@ -170,3 +173,126 @@ def vendor_total_sales(merchant_id):
...
+        return []
+
+
+    try:
+        shop_list: list = []
+        for shop in shops:
+            shop = shop_tuple_to_dict(shop)
+            shop_list.append(shop)
+            # print(shop_list)
+        return shop_list
```
Comment (zxenonx) [inline utils.py:295]: return [shop_tuple_to_dict(shop) for shop in shops]

**Review by zxenonx — COMMENTED** [review]
Thanks @Dev-wonderful 
I left a comment

File: utils.py (hunk 1 above)
Comment (Dev-wonderful) [reply utils.py:295]: Okay boss

**Pull request merged**